arrivals-example/src/main/scala/com/pagerduty/arrivals/example/ExampleGateway.scala

```diff
@@ -18,7 +18,7 @@ class ExampleGateway(
     listenPort: Int = 8080
   )(implicit actorSystem: ActorSystem,
     materializer: Materializer,
-    metrics: Metrics = NullMetrics)
+    val metrics: Metrics = NullMetrics)
```

Comment:
Prepending `val` makes `metrics` public, and therefore it can be used to satisfy `def metrics: Metrics` which was added in the various abstract controllers mixed into this class.